src/Ankimon/__init__.py

```diff
@@ -1007,47 +927,7 @@ def get_levelup_move_for_pokemon(pokemon_name, level):
             #return f"{pokemon_name} does not learn any new moves at level {level} or lower."
             return eligible_moves
 
-def split_string_by_length(input_string, max_length):
-    current_length = 0
-    current_line = []
-
-    for word in input_string.split():
-        word_length = len(word)  # Change this to calculate length in pixels
-
-        if current_length + len(current_line) + word_length <= max_length:
-            current_line.append(word)
-            current_length += word_length
-        else:
-            yield ' '.join(current_line)
-            current_line = [word]
-            current_length = word_length
 
-    yield ' '.join(current_line)
-
-def split_japanese_string_by_length(input_string, max_length):
-    max_length = 30
-    current_length = 0
-    current_line = ""
-
-    for char in input_string:
-        if current_length + 1 <= max_length:
-            current_line += char
-            current_length += 1
-        else:
-            yield current_line
-            current_line = char
-            current_length = 1
-
-    if current_line:  # Ensure the last line is also yielded
-        yield current_line
-
-def resize_pixmap_img(pixmap, max_width):
-    original_width = pixmap.width()
-    original_height = pixmap.height()
-    new_width = max_width
-    new_height = (original_height * max_width) // original_width
-    pixmap2 = pixmap.scaled(new_width, new_height)
-    return pixmap2
 
 def random_battle_scene():
     global battlescene_path

```

Comment:
by the way there're huge amount of functions that can't be moved to business.py because global variables
i afraid to touch this😕

Reply:
definetly understandable! Im trying to rewrite that to and wont to use more object based coding